src/stdio.c

```diff
@@ -4,11 +4,11 @@
 size_t printf(char *message)
 {
   size_t len = strlen(message);
-
-  for (size_t i = 0; i < len; i++)
-  {
-    terminal_putchar(message[i]);
-  }
-
+/*
+ * for ever letter in the message varable it will write to 
+ *  the display memory buffer to desplay tto the screen
+ */
+  for (size_t i = 0; i < len; i++) terminal_putchar(message[i]);
+  //after message is displayed on screen then it returns the original message car size
```

Comment:
I appreciate increasing the code clarity but these comments are really redundant and such comments should be avoided. When adding comments, at least try to make them grammatically readable.